src/style.css

```diff
@@ -0,0 +1,88 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir.ttc);
+  font-style: bold;
+  font-weight: 900;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+}
+
+.firstForm {
+  margin-top: 20px;
+  position: relative;
+}
+
+.secondForm {
+  margin: 20px 0;
+  position: relative;
+}
+
+.search {
+  font-family: inherit;
+  color: rgba(61, 78, 97, 1);
+  font-size: 16px;
+  line-height: 22px;
```

Comment:
It's better to use absolute values instead of pixel ones for line-height rule. Like 1.37 or 137%